redis/acknowledge.lua

```diff
@@ -1,8 +1,7 @@
 local zset_key = KEYS[1]
 local processed_key = KEYS[2]
 
-local worker_id = ARGV[1]
-local test = ARGV[2]
+local test = ARGV[1]
```

Comment:
It's not about Lua vs Python. Most of what is done in Lua couldn't be done in Python at all.

Those lua scripts are executed inside Redis itself.